### ISSUE TITLE
[Crash..?] Spellbook on Lecturn kicks the player from game session

### ISSUE THREAD

**Issue by NikkeiZX** [Mithion/ArsMagica2#1213]
So I'm having an issue... I'm using the 1.4.0.008 version of Ars Magica 2 (for 1.7.10) and every time I try to flip through the pages of a spell book on the lectern, it "crashes" and (for some reason) takes me to the multiplayer connect screen.... Thing is, the entire game itself doesn't crash (meaning minecraft still runs, I just simply get kicked out of the world I'm playing on and sent to the multiplayer connect screen). That said, this doesn't necessarily produce a crash log, so here's what I manage to get from the "latest" file in the "logs" folder (I duplicated this crash twice just so it's easier to spot I suppose...this is being ran in a fairly large modpack I created and am launching in Technic Launcher, if that's of any importance... also, if it matters, the crafting altar is made of Quartz with moonstone caps.. not sure if that's important information or not)

http://pastebin.com/XMewt2xf

what's even more bizarre: a friend of mine (who is one of my close friends that I shared my pack with to play test it) found that if you open the book (in creative), close the book, go into survival mode, and open the book again, it'll function properly when you turn the pages without crashing. But for some odd reason it only works sometimes o_O which is really odd...

**Comment by Mithion** [comment]
I'd need the logs from the server in order to help with this.

**Comment by NikkeiZX** [comment]
By server, do you mean as in multiplayer? Because here's the next weird thing (I forgot to mention this lol) : this is happening on single player

**Comment by NikkeiZX** [comment]
Would it help if I shared the modpack with you as well? I launch/host it through Technic Launcher. But yea, the issue (even for my friend who ran into the same problem) is happening on singleplayer

**Comment by Mithion** [comment]
Sounds like a conflict with another mod. Try removing a few at a time and see if you can isolate the mod that's problematic.   Before doing that, try just am2 and animapi to see if the issue is still there.

**Comment by NikkeiZX** [comment]
Ok. As I'm doing that, maybe there's something you can find in this log? I carried my modpack over to AT launcher and, RIGHT before I duplicated the crash, I cleared the console. So this is the code from that: http://paste.atlauncher.com/view/4c1bdadb just in case this may help at all

**Comment by Mithion** [comment]
I'd start by removing "Lost Books", that's the only thing jumping out at me there.
Seems to be throwing an error when trying to send/receive a packet (even when you're playing singleplayer it's emulating a server in the background)

**Comment by NikkeiZX** [comment]
You're right! I feel kinda dumb because I looked at the code I gave you and noticed Lost Books being referenced, so I toyed around with it and.. yea.. Without Lost Books, everything works just fine. even before reading your reply just now I figured it out haha. So yea, I showed the Mod Author for Lost Books the issue. Sorry if I wasted any of your time x_x It's just that because the issue only happened whenever I did Ars Magica stuff ,I assumed that was part of the problem :/ But yea, Sorry about that! on the other hand ,I'm glad It's not AM2 because I love this mod and would hate to have to axe it from my modpack, especially since the entire point of my modpack is to be an RPG based one lol
